packages/tailwindcss/src/utilities.ts

```diff
@@ -3524,6 +3500,9 @@ export function createUtilities(theme: Theme) {
     })
 
     functionalUtility('backdrop-opacity', {
+      // TODO: Do we want to remove this or not? Named opacity utilities (apart
+      // form like `opacity-disabled` do not make a lot of sense). Just bare
+      // values make more sense.
```

Comment:
Motivation is backwards compatibility for anyone who had something like `opacity-backdrop` defined in their config. I don't think we'd really document these theme keys going forward but we're going to have to re-introduce all this stuff for BC if we get rid of it I think.